567 beëindiging verdelging oost vlaanderen (#568)
inbo/vespa-watch: pages/en/report-nest.md

- Eradication of Asian hornet nests is is currently not compensated by the government.
- A coordinated control for Asian hornet is in preparation.

## Is eradication free of charge?

Currently, there is no coordinated Asian hornet eradication, so we do not automatically assign an eradicator for an Asian hornet nest. The government currently does not reimburse eradication costs.

Exceptions to this general rule are:

> - In **West Flanders** (except Middelkerke), all nests have been removed free of charge since August 1, 2023. If you wish to take advantage of this service, you don't need to do anything after your report on vespawatch.be. In case of an unclear photo for validation, you will be contacted to provide a clearer photo. Validated nests in this region will be forwarded to the respective exterminator. Please be aware that there may be a certain waiting period associated with this. If you want to have a nest removed quickly, you can contact a professional exterminator at your own expense.

Reviewer: ```suggestion
- In **West Flanders** (except Middelkerke), all reported nests are being removed free of charge since August 1, 2023. If you want to take advantage of this service, you don't need to do anything after reporting your sighting on vespawatch.be. If a better picture is needed to confirm it is indeed the nest of an Asian hornet, they will get in touch with you. Once the sighting is validated,  it will be forwarded to the right exterminator. Please be aware that there a bit of a wait. If you want to have a nest removed quickly, you can contact a professional exterminator at your own expense.
```

> - In **East Flanders**, from 6 November 2023, nests of the Asian hornet will no longer be controlled free of charge. You can find the procedure for East Flanders [here](https://oost-vlaanderen.be/wonen-en-leven/natuur-en-milieu/overlastsoorten/aziatische-hoornaar.html). You can contact a private exterminator yourself (overview available on the [list of eradicators](https://vespawatch.be/eradicators/)) or the [fire brigade](https://1722.be/en/index.html) if you want to have this nest removed which is at your own expense.

Reviewer: ```suggestion
- In **East Flanders**, from 6 November 2023, nests of the Asian hornet will no longer be removed for free. You can find more details [here](https://oost-vlaanderen.be/wonen-en-leven/natuur-en-milieu/overlastsoorten/aziatische-hoornaar.html). You can either hire a private exterminator yourself (overview available on the [list of eradicators](https://vespawatch.be/eradicators/)) or the [fire brigade](https://1722.be/en/index.html) if you want to have this nest removed.
```

> - In **Flemish Brabant**, specific municipalities have their own programs: [Begijnendijk](https://www.begijnendijk.be/subsidie-voor-neutralisatie-aziatische-hoornaar), [Tremelo](https://www.tremelo.be/product/3399/subsidie-bestrijding-aziatische-hoornaar), [Keerbergen](https://www.keerbergen.be/file/download/13055/A0D93C95A2B650AFF95B4DD8F1169F3C), [Boortmeerbeek](https://www.boortmeerbeek.be/subsidie-bestrijding-aziatische-hoornaar) en [Leuven](https://leuven.be/aziatische-hoornaar). More information can be found on the websites of these municipalities.

Reviewer: ```suggestion
- In **Flemish Brabant**, certain municipalities have their own programs: [Begijnendijk](https://www.begijnendijk.be/subsidie-voor-neutralisatie-aziatische-hoornaar), [Tremelo](https://www.tremelo.be/product/3399/subsidie-bestrijding-aziatische-hoornaar), [Keerbergen](https://www.keerbergen.be/file/download/13055/A0D93C95A2B650AFF95B4DD8F1169F3C), [Boortmeerbeek](https://www.boortmeerbeek.be/subsidie-bestrijding-aziatische-hoornaar) en [Leuven](https://leuven.be/aziatische-hoornaar). More information can be found on the websites of these municipalities.
```
